dependencies/include/asio/detail/impl/win_iocp_io_service.ipp

```diff
@@ -462,11 +466,18 @@ DWORD win_iocp_io_service::get_gqcs_timeout()
   osvi.dwOSVersionInfoSize = sizeof(osvi);
   osvi.dwMajorVersion = 6ul;
 
-  const uint64_t condition_mask = ::VerSetConditionMask(
-      0, VER_MAJORVERSION, VER_GREATER_EQUAL);
 
+  
+    // @third party code Studio Gobo
+    // Hack to get project compiling on UWP as ::TerminateThread is not available on that platform
+    // https://docs.microsoft.com/en-us/windows/desktop/api/processthreadsapi/nf-processthreadsapi-terminatethread
+#if false
```

Comment:
Do we want to disable this on all platforms, or just the one for which it doesn't work?